src/components/footer.css

```diff
@@ -0,0 +1,115 @@
+.footer {
+  background-color: #000;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.footer-container {
+  max-width: calc(1440px + 2 * 84px);
+  width: calc(1440px + 2 * 84px);
+  padding: 0 84px;
+  height: auto;
+}
+
+.footer-content-container {
+  padding: 90px 0;
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  height: auto;
+}
+
+.footer-content-container__left,
+.footer-content-container__links {
+  display: flex;
+}
+
+.footer-content-container__logo {
+  background-image: url("../images/logo-small.svg");
+  margin-right: 126px;
+  width: 88px;
+  background-repeat: no-repeat;
+  background-position: left top;
+}
+
+.footer-content-container__links-list {
+  margin-right: 148px;
+  font-family: "Visuelt Bold", sans-serif;
```

Comment:
You should be able to use the global variable here and below in place of this. Also, let's make a note after these two branches are complete to setup our base font styles for the project, that way we won't need to update individual styles as often.